packages/cli/test/scripts/push.test.js

```diff
@@ -266,7 +266,7 @@ contract('push script', function([_, owner]) {
       if (!this.networkFile.appAddress) return;
       const app = await App.fetch(this.networkFile.appAddress);
       const packageInfo = await app.getPackage(libName)
-      packageInfo.version.should.be.semverEqual(libVersion)
+      packageInfo.version.map(Number).should.be.semverEqual(libVersion)
```

Comment:
We could move the `.map(Number)` inside the semverEqual helper actually. But I'm fine with leaving this as is.